tools/syz-bq.sh

```diff
@@ -92,7 +92,7 @@ echo The latest commit as of $to_date is $base_commit.
 # echo Temp dir $base_dir deleted.
 
 from_date=$(date -d "$to_date - $duration days" +%Y-%m-%d)
-sessionID=$(uuidgen)
+sessionID=$(cat /proc/sys/kernel/random/uuid)
```

Comment:
FWIW there's also https://linux.die.net/man/1/mktemp that should replace both sessionID generation and `mkdir`.

Reply:
I also need it in the query.
Mktemp way requires id parsing from the command output.